src/components/collapse/Collapse.svelte

```diff
@@ -0,0 +1,96 @@
+<div class="{baseClasses}">
+  <slot />
+</div>
+
+<script>
+  import { setContext, getContext, createEventDispatcher } from "svelte";
+  import { writable } from "svelte/store";
+  import classNames from "classnames";
+  import { RightOutlined } from "@/components/icons";
+  import { CONFIG_KEY, configProvider } from "@/provider/config-provider";
+
+  const dispatch = createEventDispatcher();
+
+  // ********************** Props **********************
+
+  // Sets the default open panel(s)
+  export let defaultActiveKey = "";
+  // Manually override the keys for more control of what opens and when
+  export let activeKey = null;
+  // Accordion mode only allows one panel open at a time
+  export let accordion = false;
+  // Borderless version
+  export let borderless = false;
+  // Custom expand icon
+  export let expandIcon = RightOutlined;
+  // Custom class for the Collapse wrapper
+  export let className = "";
+  // Icon position can be left or right
+  export let expandIconPosition = "";
+
+  // ********************** /Props **********************
+
+  // Class name for the icon positon - set below
+  let iconPositonClassName = "";
+  // Classes for the collapse wrapper
+  let baseClasses = "";
+
+  const config = getContext(CONFIG_KEY) || configProvider();
+  const { getPrefixCls, direction } = $config;
+  const prefixCls = getPrefixCls("collapse");
+
+  const collapseStore = writable({
+    activeKey: [],
+    isAccordion: false,
+    expandIcon,
+    usingCustomActiveKey: false
+  });
+  $: collapseStore.set({
+    activeKey:
+      activeKey ||
+      (typeof defaultActiveKey === "object"
+        ? defaultActiveKey
+        : [defaultActiveKey]),
+    isAccordion: !!accordion,
+    expandIcon,
+    usingCustomActiveKey: activeKey
+  });
+  setContext("collapseStore", collapseStore);
```

Comment:
can we move this above `$: collapseStore.set` (this is just me being picky 😂)

Reply:
😆 This makes sense though! I'll move it 💯